constants/Colors.ts

```diff
@@ -3,24 +3,4 @@
  * There are many other ways to style your app. For example, [Nativewind](https://www.nativewind.dev/), [Tamagui](https://tamagui.dev/), [unistyles](https://reactnativeunistyles.vercel.app), etc.
  */
 
-const tintColorLight = '#0a7ea4';
-const tintColorDark = '#fff';
 
-export const Colors = {
```

Comment:
Doesn't this deletion breaks anything?

Reply:
I think that this constant is used somewhere, but am seeing refactoring of these usage.

Reply:
> Doesn't this deletion breaks anything?

I dunno yet. Will run it again in a few minutes

Reply:
> Doesn't this deletion breaks anything?

Oui, il y a un problème effectivement puisque ces couleurs étaient utilisées